views/home/doctorHome.php

```diff
@@ -27,11 +27,274 @@
 
       <a href="../admin/roster.php">Look at todays Roster</a>
-      
+  
     </section>
 
+
+    <section>
+      <p>If you would like to filter the results fill in a field only one field at a time.</p>
+
+      <form  action="doctorHome.php" method="post">
+
+        <label>Date:
+          <input type="text" name="date">
+        </label>
+
+        <label>First Name:
+          <input type="text" name="Fname">
+        </label>
+
+        <label>Last Name:
+          <input type="text" name="Lname">
+        </label>
+
+        <label>Comment:
+          <input type="text" name="Comment">
+        </label>
+
+        <label>Morning Medication:
+          <input type="text" name="mmed">
+        </label>
+
+        <label>Afternoon Medication
+          <input type="text" name="amed">
+        </label>
+
+        <label>Night Medication
+          <input type="text" name="nmed">
+        </label>
+
+        <input type="submit" name="Submit" value="Submit">
+
+      </form>
+
+      <?php
+
+      if (isset($_POST['Submit'])) {
+
+        // Getting post requests and naming column names
+        $date = $_POST['date'];
+        $Fname = $_POST['Fname'];
+        $Lname = $_POST['Lname'];
+        $Comment = $_POST['Comment'];
+        $Mmed = $_POST['mmed'];
+        $Amed = $_POST['amed'];
+        $Nmed = $_POST['nmed'];
+
+        $day = 'day';
+        $Last = 'Lname';
+        $first = 'Fname';
+        $com = 'Comment';
+        $morn = 'morning_med';
+        $aft = 'afternoon_med';
+        $night = 'night_med';
+        $filter = '';
+        $column = '';
+
+        $id = intval($_SESSION['id']);
+
+        $db_link = mysqli_connect("localhost", "root", "", "retire");
+
+        if ($db_link == false) {
+          die("ERROR: Could not connect. " . mysqli_connect_error());
+        }
+
+        // Check only one field can be filled
+        // then setting proper variables.
+        if (!empty($date)) {
+          $filter = $date . '%';
+          $column = $day;
+        }
+        else {
+
+          if (!empty($Fname)) {
+            $filter = $Fname . '%';
+            $column = $first;
+          }
+
+          else {
+
+              if (!empty($Lname)) {
+                $filter = $Lname . '%';
+                $column = $Last;
+              }
+
+              else {
+
+                if (!empty($Comment)) {
+
+                  $filter = $Comment . '%';
+                  $column = $com;
+                }
+
+                else {
+
+                  if (!empty($Mmed)) {
+
+                    $filter = $Mmed . '%';
+                    $column = $morn;
+
+                  }
+
+                  else {
+
+                    if (!empty($Amed)) {
+
+                      $filter = $Amed . '%';
+                      $column = $aft;
+
+                    }
+
+                    else {
+
+                      if (!empty($Nmed)) {
```

Comment:
this whole section isn't wrong, but just a reminder that `elseif` does exist and would save a lot of lines and indentations